ArcGIS-toolbox/ASF_Tools.pyt

```diff
@@ -762,12 +782,11 @@ class RGBDecomp(object):
         indir = parameters[0].valueAsText
         scale = parameters[1].valueAsText
         pol = parameters[2].valueAsText
-        rb_thresh_db = parameters[3].valueAsText
```

Comment:
This is definitely a required variable.
Ah, I see. I hard-coded in a value when I should have used this input variable on line 840. 
Instead of -2.4 it should be rb_thresh_db/10.